templates/partials/expense-items.hbs

```diff
@@ -7,7 +7,7 @@
           <td style="width: 66px; border-bottom: 1px dotted rgb(196, 199, 204); padding: 12px 0;">
             <div style="border: 1px solid #dcdee0; width: 48px; height: 48px; padding: 4px;">
               <a href="{{this.url}}">
-                <div class="preview" style="width: 100%; height: 100%; z-index:0; background-size: contain; background-repeat: no-repeat; background-position: center; background-image: url('{{resizeImage this.url width=640}}');"></div>
+                <div class="preview" style="width: 100%; height: 100%; z-index:0; background-size: contain; background-repeat: no-repeat; background-position: center; background-image: url('{{this.url}}?thumbnail');"></div>
```

Comment:
This won't work: most email clients proxy the images to prevent tracking. I suggest:
1. Redesigning the email to remove previews
2. OR, when fetching thumbnails from the controller as unauthenticated/unauthorized, return a generic file icon.

Reply:
Replaced with a generic placeholder icon. The controller would return the same icon for every resource (existing or not, regardless of the mime type) to prevent exposing any info to unauthenticated requests.